Gemfile.lock

```diff
@@ -417,6 +417,3 @@ DEPENDENCIES
   uglifier (~> 2.5.3)
   unicode (= 0.4.4)
   unicorn (~> 4.8.3)
-
-BUNDLED WITH
```

Comment:
No, I just ran bundle install as usual and it removed it.  I just checked and I had an old version of bundler (1.8.x), so I updated to 1.10.6 and it's adding the bundler version to the end of the file.  I'll push another commit that adds that back.